src/components/introduction-card/IntroductionCard.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+interface IntroductionCard {
+    id: number
+    title: string
+    content: string
+    badgeText: string
+    badgeStyle: string
+    imgUrl: string
+    btnText: string
+    bgColor: string
+  }
+
+const props = defineProps<{
+  introductionCard: IntroductionCard
+}>()
+</script>
+
+<template>
+  <div class="intro-card-container">
+    <q-card class="intro-card" flat :class="introductionCard.bgColor">
+      <q-card-section class="intro-card-wrapper" horizontal>
+        <q-card-section class="main-container no-padding">
+          <div class="wrapper-main flex">
+            <q-badge class="badge" :class="introductionCard.badgeStyle">{{ props.introductionCard.badgeText }}</q-badge>
+            <div>
+              <div class="title q-mt-sm q-mb-xs">{{ props.introductionCard.title }}</div>
+              <div class="content">{{ props.introductionCard.content }}</div>
+            </div>
+          </div>
+
+          <q-btn class="btn text-capitalize">
+            <div class="text">{{ props.introductionCard.btnText }}</div>
+            <q-icon class="icon" name="arrow_forward" />
+          </q-btn>
+        </q-card-section>
+
+        <q-card-section class="image-container col-5 flex flex-center no-padding">
+          <img
+            class="img-promote rounded-borders"
+            :src="introductionCard.imgUrl"
+            alt="promote-man.png"
+          />
+        </q-card-section>
+      </q-card-section>
+    </q-card>
+  </div> 
+</template>
+
+<style lang="sass" scoped>
```

Comment:
I think we use scss to easier to code,  don't you?

Reply:
Why dont you ask me at the beginning? But it's okay, let me fix it